cmake/Modules/FindRapidJson.cmake

```diff
@@ -26,4 +26,4 @@ if (NOT RAPID_JSON_INCLUDE_DIR OR NOT RAPID_JSON_LIBRARIES)
 endif ()
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(RAPID_JSON DEFAULT_MSG RAPID_JSON_INCLUDE_DIR )
\ No newline at end of file
+find_package_handle_standard_args(RapidJson DEFAULT_MSG RAPID_JSON_INCLUDE_DIR )
```

Comment:
Fixes the following Compiler warning:
```
CMake Warning (dev) at .../FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (RAPID_JSON)
  does not match the name of the calling package (RapidJson).  This can lead
  to problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
  ```
  
The variables automatically set by this call (Eg: RAPID_JSON_FOUND) aren't used - from what I can see - so this change shouldn't have any effect on the build. I tested it with the windows build obviously, but, looking through the build pipeline I didn't see any references that would be broken by this.

Reply:
will take a look on Linux